vllm/attention/backends/blocksparse_attn.py

```diff
@@ -87,6 +87,8 @@ def __post_init__(self):
 
 class BlocksparseFlashAttentionBackend(AttentionBackend):
 
+    use_output: bool = True
```

Comment:
is it true?

Reply:
While base on previous logic, it's true to keep the same as before. See L95. 

While it should be False indeed after reading your PR https://github.com/vllm-project/vllm/pull/10822